tests/s390x_tests/consoletest_KERNEL_btrfs.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE’s openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+# Copyright (C) 2018 IBM Corp.
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary:  Based on consoletest_setup.pm (console test pre setup, stopping and disabling packagekit, install curl and tar to get logs and so on)
+# modified for running the testcase KERNEL_btrfs on s390x.
+
+use base "s390base";
+use testapi;
+use utils;
+use strict;
+
+sub run {
+    my $self = shift;
+    $self->copy_testsuite('KERNEL_btrfs');
+    $self->execute_script('btrfs.eckd.sh', '"xxxx xxxx xxxx xxxx xxxx xxxx"', 1200);
+}
+
+sub post_fail_hook {
+    my $self = shift;
+   $self->export_logs();
```

Comment:
somehow logs for this test are not uploaded... not sure why exactly - other logs are uploaded correctly

Reply:
It works in my env. Please provide test case logs from the test system.